molecule/affinity-tolerations-resources-test/converge.yml

```diff
@@ -61,6 +61,8 @@
         some.test.io/keepCamelCaseFOO: camelCaseValueFOO
       new_service_annotations:
         another.test.io/keepCamelCaseBAR: camelCaseValueBAR
+      new_pod_labels:
+        some.test.label/labelKeepCamelCaseFOO: labelCamelCaseValueFOO
```

Comment:
Maybe this should be BAZ so as not to think there is a link between this and some.test.io ?